.changeset/kind-cats-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-user-settings': minor
```

Comment:
This should just be a `patch` change I think :)

Reply:
@benjdlambert Of course, you're right 😄 
Updated to patch